analytics.go

```diff
@@ -190,7 +190,6 @@ func (c *client) Enqueue(msg Message) (err error) {
 		m.Type = "alias"
 		m.MessageId = makeMessageId(m.MessageId, id)
 		m.OriginalTimestamp = makeTimestamp(m.OriginalTimestamp, ts)
-		m.SentAt = m.OriginalTimestamp
```

Comment:
Let's still set the SentAt to the original timestamp to help in payload size calculation before finalizing the events for the batch.

Reply:
I made the requested changes [666ce95](https://github.com/rudderlabs/analytics-go/pull/27/commits/666ce9581f7e1cf64de68864dd50eeceb30407d0)